cmd/crio/config.go

```diff
@@ -4,6 +4,7 @@ import (
 	"os"
 	"text/template"
 
+	"github.com/cri-o/cri-o/pkg/criocli"
```

Comment:
Can we call this package only `cli` to reduce the noise a bit? :innocent:

Reply:
`cli` would conflict with `github.com/urfave/cli`, which is imported both in main and criocli. I'd have to import `criocli "github.com/cri-o/cri-o/pkg/cli`, which i find noiser

Reply:
I agree :+1: